lib/sql-string.js

```diff
@@ -35,6 +35,11 @@ function escape(val, timeZone, dialect, format) {
     break;
   }
 
+  // moment
+  if (val.toDate) {
```

Comment:
Why? Anything that has a toDate method can obviously be converted to a Date. Checking isValid can still be done.